clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/client/DockerRegistryClient.groovy

```diff
@@ -391,7 +391,7 @@ class DockerRegistryClient {
       DockerBearerToken dockerToken = tokenService.getToken(target)
       String token
       if (dockerToken) {
-        token = "Bearer ${dockerToken.access_token ?: dockerToken.token}"
+        token = "Bearer ${(dockerToken.bearer_token ?: dockerToken.token) ?: dockerToken.access_token}"
```

Comment:
@lwander I'm not super familiar with Groovy's Elvis operators, does this make sense?